processing/src/test/java/org/apache/druid/java/util/common/GranularityTest.java

```diff
@@ -1011,7 +1011,6 @@ public void testGranularitiesFinerThanAll()
             Granularities.YEAR,
             Granularities.QUARTER,
             Granularities.MONTH,
-            Granularities.WEEK,
```

Comment:
For completeness, could we also add a test `testGranularitiesFinerThanWeek()` that does include the week granularity when it's requested?